games-server/minecraft-server/minecraft-server-1.14.4.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_COMMIT="3dc3d84a581f14691199cf6831b71ed1296a9fdf"
+
+inherit readme.gentoo-r1 java-pkg-2 user
+
+DESCRIPTION="The official server for the sandbox video game"
+HOMEPAGE="https://www.minecraft.net/"
+SRC_URI="https://launcher.mojang.com/v1/objects/${EGIT_COMMIT}/server.jar -> ${P}.jar"
+
+LICENSE="Mojang"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-misc/screen
+	>=virtual/jre-1.8
+"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}"
+
+pkg_setup() {
+	enewgroup minecraft
```

Comment:
This needs to be handled via the GLEP-81 method now. You've probably seen enough example messages in gentoo-dev (new acct-*/ ebuilds needs to be reviewed there first).

Reply:
```Following the acceptance of this GLEP, all new users and groups must be created via user/group packages as defined in this GLEP. The old method may still be used for existing users/groups, in existing packages.```

This says, it's not mandatory for existing users?

FWICS, I would anyway to submit frist to ml? So my proposal, let's merge this the old way, so I can prepare for all of my packages, which adds an user/group, patches for ml for review?

Reply:
I understood that so existing ebuilds need not to be updated, but every new ebuild should. @mgorny could you comment for clarification?

And if you meant preparing for ALL of your packages, I think it's wiser to do it little-by-little when specifically updating those user/guid ebuilds.

Reply:
Please have a look, I've updated to ```acct-*``` packages, since the got approved on ml :-)